docs/basic-features/image-optimization.md

```diff
@@ -32,180 +22,128 @@ Now, you can define the `src` for your image (either local or remote).
+    />
   )
 }
 ```
 
-> Learn more about the [sizing requirements](#image-sizing) in `next/image`.
-
-### Domains
-
-Sometimes you may want to optimize a remote image, but still use the built-in Next.js Image Optimization API. To do this, leave the `loader` at its default setting and enter an absolute URL for the Image `src` prop.
-
-To protect your application from malicious users, you must define a list of remote hostnames you intend to use with the `next/image` component.
-
-> Learn more about [`remotePatterns`](/docs/api-reference/next/image.md#remote-patterns) configuration.
-
-### Loaders
-
-Note that in the [example earlier](#remote-images), a partial URL (`"/me.png"`) is provided for a remote image. This is possible because of the loader architecture.
-
-A loader is a function that generates the URLs for your image. It modifies the provided `src`, and generates multiple URLs to request the image at different sizes. These multiple URLs are used in the automatic [srcset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset) generation, so that visitors to your site will be served an image that is the right size for their viewport.
-
-The default loader for Next.js applications uses the built-in Image Optimization API, which optimizes images from anywhere on the web, and then serves them directly from the Next.js web server. If you would like to serve your images directly from a CDN or image server, you can write your own loader function with a few lines of JavaScript.
-
-You can define a loader per-image with the [`loader` prop](/docs/api-reference/next/image.md#loader), or at the application level with the [`loaderFile` configuration](https://nextjs.org/docs/api-reference/next/image#loader-configuration).
-
-### Priority
-
-You should add the `priority` property to the image that will be the [Largest Contentful Paint (LCP) element](https://web.dev/lcp/#what-elements-are-considered) for each page. Doing so allows Next.js to specially prioritize the image for loading (e.g. through preload tags or priority hints), leading to a meaningful boost in LCP.
-
-The LCP element is typically the largest image or text block visible within the viewport of the page. When you run `next dev`, you'll see a console warning if the LCP element is an `<Image>` without the `priority` property.
-
-Once you've identified the LCP image, you can add the property like this:
-
-```jsx
-import Image from 'next/image'
-
-export default function Home() {
-  return (
-    <>
-      <h1>My Homepage</h1>
-      <Image
-        src="/me.png"
-        alt="Picture of the author"
-        width={500}
-        height={500}
-        priority
-      />
-      <p>Welcome to my homepage!</p>
-    </>
-  )
-}
+Next, define a list of supported URL patterns you want to allow optimizing images from. Be as specific as possible to prevent malicious usage. For example, the following configuration will only allow images from a specific AWS S3 bucket:
```

Comment:
```suggestion
Then, define a list of remote URL patterns you want to allow optimizing images from. Be as specific as possible to prevent malicious usage. For example, the following configuration will only allow images from a specific AWS S3 bucket:
```